### ISSUE TITLE
kubernetes eventrouter had more data

### ISSUE THREAD

**Issue by data-dude** [resmoio/kubernetes-event-exporter#51]
I noticed that kubernetes eventrouter would get data when the verb == UPDATED and ADDED where kubernetes event exporter only gets ADDED data.  What is the verb?  I'm not sure.  It's not reason.  I've started looking at the kubernetes eventrouter code but i'm not sure what it is.   Does anyone know what this is?  is verb == UPDATED data important?

**Comment by Evedel** [comment]
Hey @data-dude,

That code change was not released yet, so unless you are building from sources, it should not be what is affecting you. Likely you have the same issues [as in this PR](https://github.com/resmoio/kubernetes-event-exporter/pull/37), can you please check if increasing `throttlePeriod` will help you?

**Comment by data-dude** [comment]
@Evedel I thought throttle was deprecated?  I added kubeQPS: 100 and kubeBurst: 500 and maxEventAgeSeconds: 6000  and I'm still seeing drops.  I have a cronjob that runs every 2 minutes so i can clearly see when the event exporter misses an event.  My configuration is just dumping everything into Elastic.

**Comment by Evedel** [comment]
Yep, thanks for checking, there seems to be a discrepancy between the codebase and the latest release at the moment (ie the doc on the main is for the version that was not released yet).

fyi @mustafaakin it may be worth it to revert the readme if you are not comfortable with a new release yet
